src/bin/observable.ts

```diff
@@ -217,7 +217,7 @@ try {
         );
       }
     } else {
-      console.error(`\n${red("Unexpected error:")} ${error.message}`);
+      console.error(`\n${red("Unexpected error:")} ${(error as any).message}`);
```

Comment:
This is needed because I upgraded TypeScript (which was needed because of import attributes).